test/unit/util/hash-project-loader-hoc.test.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import ProjectLoaderHOC from '../../../src/lib/project-loader-hoc.jsx';
+import HashParserHOC from '../../../src/lib/hash-parser-hoc.jsx';
+import storage from '../../../src/lib/storage';
+import {mount} from 'enzyme';
+
+jest.mock('react-ga');
+
+describe('Hash/ProjectLoaderHOC', () => {
+    test('when there is no project data, it renders null', () => {
+        const Component = ({projectData}) => <div>{projectData}</div>;
+        const WrappedComponent = HashParserHOC(ProjectLoaderHOC(Component));
+        window.location.hash = '#winning';
+        const originalLoad = storage.load;
+        storage.load = jest.fn(() => Promise.resolve(null));
+        const mounted = mount(<WrappedComponent />);
+        storage.load = originalLoad;
+        window.location.hash = '';
+        const mountedDiv = mounted.find('div');
+        expect(mountedDiv.exists()).toEqual(false);
+    });
+
+    test('when there is no hash, it loads the default project', () => {
```

Comment:
I think we should split this in two:
1. When there is no hash, `HashParserHOC` passes `0` as the `projectId` prop
2. If `0` is passed to  `ProjectLoaderHOC` it loads the default project